macros/AddonSidebar.ejs

```diff
@@ -393,6 +393,104 @@ var text = mdn.localStringMap({
     'Blog': '부가 기능 블로그',
     'Forums': '부가 기능 포럼',
     '#Contact_us': '연락하기'
+  },
+  'zh-CN': {
+    'WebExtensions': '浏览器扩展程序',
+    'WebExtensions#Getting_started': '开始',
+    'WebExtensions/What_are_WebExtensions': '什么是扩展程序?',
+    'WebExtensions/Your_first_WebExtension': '你的第一个扩展程序',
+    'WebExtensions/Your_second_WebExtension': '你的第二个扩展程序',
+    'WebExtensions/Anatomy_of_a_WebExtension': '详解扩展程序,
```

Comment:
This is why the Travis CI is failing. A SyntaxError in your `.ejs`` file.

Reply:
Thanks,you're right.I'm too careless~I‘ve fixed them and passed now~